contentcuration/contentcuration/static/js/edit_channel/import/vuex/importModule.js

```diff
@@ -0,0 +1,25 @@
+var mutations = require('./importMutations');
+var actions = require('./importActions');
+var getters = require('./importGetters');
+
+module.exports = {
+  namespaced: true,
+  state: {
+    // An array of JSON-ified ContentNode Models, which is updated
+    // whenever a user selects a Node to import
+    itemsToImport: [],
+    // start | show_warning | finish | null
+    importStatus: null,
```

Comment:
I like how component [prop validation](https://vuejs.org/v2/guide/components.html#Prop-Validation) removes some of this boilerplate, and defines the type names and validator functions in a declarative way.

I'm kind of surprised they didn't design the vuex `state` object to look and behave more like the vue `props` object.